Makefile

```diff
@@ -356,13 +358,23 @@ redeploy_nodes: destroy_nodes deploy_nodes
 
 redeploy_nodes_with_install: destroy_nodes deploy_nodes_with_install
 
+############
+# Operator #
+############
+
+clear_operator:
+	DISKS="${OPERATOR_DISKS}" ./assisted-service/deploy/operator/destroy.sh
+
+deploy_assisted_operator: clear_operator
+	NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) ENABLE_KUBE_API=true DEPLOY_TARGET=operator ./scripts/deploy_assisted_service.sh
```

Comment:
We still need ``ENABLE_KUBE_API=true``?

Reply:
I set it for @danielerez code on https://github.com/openshift/assisted-test-infra/blob/0e7266436bd431b84c011c58fd1eddcb39a786da/scripts/deploy_assisted_service.sh#L21-L28 to run. But I'll extend the condition to work also with `DEPLOY_TARGET=operator` instead.